.taskcluster.yml

```diff
@@ -75,197 +74,222 @@ tasks:
               || (tasks_for == "github-push" && head_branch == "refs/heads/main")
               || (isPullRequest && pullRequestAction in ["opened", "reopened", "synchronize"])
           then:
-              taskId: {$if: 'tasks_for != "action"', then: '${ownTaskId}'}
-              taskGroupId:
-                  $if: 'tasks_for == "action"'
-                  then:
-                      '${action.taskGroupId}'
-                  else:
-                      '${ownTaskId}'  # same as taskId; this is how automation identifies a decision task
-              schedulerId: '${trustDomain}-level-${level}'
-              created: {$fromNow: ''}
-              deadline: {$fromNow: '1 day'}
-              expires: {$fromNow: '1 year 1 second'}  # 1 second so artifacts expire first
-              metadata:
-                  $merge:
-                      - owner: "${ownerEmail}"
-                        source: "${repoUrl}/raw/${head_sha}/.taskcluster.yml"
-                      - $switch:
-                            'tasks_for == "github-push" || isPullRequest':
-                                name: "Decision Task"
-                                description: 'The task that creates all of the other tasks in the task graph'
-                            'tasks_for == "action"':
-                                name: "Action: ${action.title}"
-                                description: |
-                                    ${action.description}
+              $let:
+                  level:
+                      $if: 'tasks_for in ["github-push", "cron", "action"] && repoUrl == "https://github.com/mozilla/firefox-translations-training"'
```

Comment:
No, I think this would apply to PRs created from anywhere come to think of it....(head repo is the one containing the PR branch and base is the repo the PR is being opened against, I think?).

Seems like this is a longstanding oversight...

Reply:
Rereading, and I think you're right -- actions on PRs that were creating from a branch on the canonical repo would get level 3 for actions. (head repo would be the canonical one only for those - it would be a different repo for PRs like this)

Regardless, I agree that it's not ideal that actions running against a PR get level 3, when the rest of the jobs are level 1. Do we have a way to address that though? I'm not sure we have any information about PR vs. push when we render this for `task_for = "action"`...

Reply:
Yeah I don't think there is. Maybe we could derive the level from the Decision task and create a new `actionLevel` context here or something:
https://hg.mozilla.org/ci/ci-configuration/file/tip/build-decision/src/build_decision/util/trigger_action.py#l126

I'm unsure if this is a big deal or a nothing burger.

Reply:
Well...you still need scopes to do anything interesting or sensitive, so I don't think it's a security issue. I think it's mostly a confusion thing, unless I'm missing something.